### ISSUE TITLE
Shimmie based boards always? return status code 200, breaking api detection

### ISSUE THREAD

**Issue by particularCircumstances** [tjg1/nori#88]
Shimmie based boards like paheal return a status code 200 even if the api end point doesn't exist. That breaks the api detection for shimmie boards (the danbooru api is selected instead).
I propose a patch that checks the content-type during api detection as well.
[Shimmie_detect_patch.txt](https://github.com/tjg1/nori/files/751789/Shimmie_detect_patch.txt)

**Comment by tjg1** [comment]
What shimmie sites in particular are you talking about? It works fine with Dollbooru.

It could be an issue with the site not having proper rewrite rules for "nice URLs".

**Comment by particularCircumstances** [comment]
You are right it seems to be only a problem with rule34.paheal.net. My guess would be that a wrong status code is more likely than a wrong content-type. But checking the returned body content is probably the only way to be sure :(
Nice app by the way :)
